app/src/main/java/com/example/lastfm/LastFMClient.java

```diff
@@ -11,7 +11,13 @@
 public interface LastFMClient {
 
     @GET("?method=chart.gettopartists")
-    Single<List<Artist>> getArtists(@Query("limit")int numberArtists, @Query("api_key")String key, @Query("format")String format);
+    Single<List<Artist>> getArtists(@Query("limit")int numberArtists,
+                                    @Query("api_key")String key,
+                                    @Query("format")String format);
+    @GET("?method=artist.getinfo")
+    Single<ArtistInfo> getArtistInfo(@Query("artist")String name,
+                                     @Query("api_key")String key,
+                                     @Query("format")String format);
```

Comment:
remove params (api_key and format) from methods and create interceptor for adding them to requests. See the task "Interceptors" in Trello

Reply:
Added interceptor